products/macos.md

```diff
@@ -10,20 +10,27 @@ releaseLabel: "macOS __RELEASE_CYCLE__ (__CODENAME__)"
 auto:
 -   custom: true
 releases:
+-   releaseCycle: "13"
+    codename: "Ventura"
+    eol: false
+    link: 
```

Comment:
Let's use the same link, since Apple will probably use the same.

Reply:
Nevermind, clicked through the links - thought we were linking to <https://support.apple.com/en-in/HT201222>, which is same for all versions.